geode-assembly/src/test/resources/expected-pom.xml

```diff
@@ -19,8 +19,8 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.geode</groupId>
   <artifactId>apache-geode</artifactId>
-  <version>1.8.0-SNAPSHOT</version>
-  <packaging>pom</packaging>
+  <version>1.9.0-SNAPSHOT</version>
+  <packaging>tgz</packaging>
```

Comment:
Is this a result of our build changes that this is now TGZ or did I screw something up?